### DISCUSSION TITLE
How can i remove the upload throttle?

### DISCUSSION THREAD

**Discussion by Ryuuji159** [livewire/livewire#7930] [1. Help]
Im implementing a way to upload large files, but I have to add a delay between chunks because the upload endpoint has a throttle that starts to throw an error if I upload too fast.
How can i disable it?

**Comment by joshhanley** [comment]
@Ryuuji159 you can see here the throttle middleware being applied

https://github.com/livewire/livewire/blob/7e7d638183b34fb61621455891869f5abfd55a82/src/Features/SupportFileUploads/FileUploadConfiguration.php#L99-L102

But if there is a config option in `livewire.temporary_file_upload.middleware` then it will use that instead.

https://github.com/livewire/livewire/blob/7e7d638183b34fb61621455891869f5abfd55a82/config/livewire.php#L70

So maybe you could just change the throttle value to suit your needs or maybe experiment with setting the config to an empty string (or some other middleware that does nothing) to bypass it.

Hope this helps!
